newbius/modules/filestore/locals.tf

```diff
@@ -4,6 +4,7 @@ locals {
       jail                 = "jail"
       controller_spool     = "controller-spool"
       jail_submount_prefix = "jail-submount"
+      accounting_storage   = "accounting-storage"
```

Comment:
I'd rather delete all kinds of `storage` suffixes from accounting. Look what I mean:
The resource we refer to is actually named Shared Filesystem in Compute. The term `filestore` is kinda unofficial, but quite meaningful - it's a storage for files. So, it looks awkward to name file storage with name containing `storage`. It's like naming a cat `Basil, the cat`, rather than just `Basil`.
So, if we deal with storage for accounting inside entities like `filestore`, `volume`, etc., it's already clear that we work with some filestore, and we can just skip `storage` in its name.